### PR TITLE
fix avx512-s8-dequant and asym related bug

### PR DESCRIPTION
**Pull request by zhewang1-intc**
## Type of Change

feature or bug fix or documentation or others
API changed or not

## Description

detail description 
Issues: xxx

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

**Comment by airMeng** [comment]
shall we set up CI for bestla separately? @VincyZhang @luoyu-intel

**Comment by luoyu-intel** [comment]
> shall we set up CI for bestla separately? @VincyZhang @luoyu-intel

Good point! We'd better reset all error thresholds, like int4_err and fp4_err, before we enable bestla ci.

File: bestla/bestla/kernel_avx2.h [hunk 1]
```diff
@@ -161,15 +161,14 @@ BTLA_CODE dequant_kblock_s8_fp_fwd(int8_t* srcptr, _DST_T* dstptr, int row, int
       auto s8_ymm_v = _mm_loadl_epi64(reinterpret_cast<__m128i*>(srcptr + i * ld_src + j));
       auto s32_ymm_v = _mm256_cvtepi8_epi32(s8_ymm_v);
       if constexpr (WITH_ZP) {
-        s32_ymm_v = _mm256_sub_epi32(
-            s32_ymm_v,
-            _mm256_cvtepi8_epi32(_mm_loadl_epi64(reinterpret_cast<__m128i*>(zero_points + kpos * NPad + j))));
+        auto zp_ymm =
+            _mm256_cvtepi8_epi32(_mm_loadl_epi64(reinterpret_cast<__m128i*>(zero_points + kpos * NPad + j / PACK_ROW)));
+        if constexpr (PACK_ROW == 4) zp_ymm = _mm256_permutevar8x32_epi32(zp_ymm, packrow4_permute_idx);
```
Comment (luoyu-intel) [inline bestla/bestla/kernel_avx2.h:166]: how about PACK_ROW==2?

Comment (zhewang1-intc) [reply bestla/bestla/kernel_avx2.h:166]: PACK_ROW==2 only needed on compute_type==bf16?
AVX2 dosen't support this compute_type.

Comment (luoyu-intel) [reply bestla/bestla/kernel_avx2.h:166]: make sense

**Review by luoyu-intel — APPROVED** [review]

**Comment by hshen14** [comment]
> > shall we set up CI for bestla separately? @VincyZhang @luoyu-intel
> 
> Good point! We'd better reset all error thresholds, like int4_err and fp4_err, before we enable bestla ci.

@VincyZhang @zhewang1-intc

**Comment by VincyZhang** [comment]
> > > shall we set up CI for bestla separately? @VincyZhang @luoyu-intel
> > 
> > 
> > Good point! We'd better reset all error thresholds, like int4_err and fp4_err, before we enable bestla ci.
> 
> @VincyZhang @zhewang1-intc

@zhewang1-intc  Will set error thresholds and add UT
@VincyZhang Will prepare Azure and setup CI to trigger UT

**Review by airMeng — APPROVED** [review]

**Comment by zhewang1-intc** [comment]
let's merge this pr first and set CI in next pr.

**Pull request merged**